composer.json

```diff
@@ -18,12 +18,12 @@
     "require": {
         "php": ">=7.0.0",
         "swiftmailer/swiftmailer": "^6.0.1",
-        "symfony/dependency-injection": "~2.7|~3.0",
+        "symfony/dependency-injection": "~2.7|~3.4@dev",
```

Comment:
We should probably allow 4.0 deps for all deps as well, perhaps something to do in another PR before merging this one?

Reply:
If that's OK, yes.

Reply:
Yep, if you can create a PR and check that all tests still pass with 4.0, that would be great.

Reply:
lower bound is 3.x then right? or are we aiming for `2.7|3.4@dev|4.0@dev` something like that?

Reply:
indeed, we should make it work for 2.7, 3.4+ and 4.0+